commands/pull.js

```diff
@@ -30,6 +31,20 @@ function exec (cmd, opts = {}) {
   }
 }
 
+function spawn (cmd) {
+  const {spawn} = require('child_process')
+  return new Promise((resolve, reject) => {
+    let result = ''
+    let psql = spawn(cmd, [], {encoding: 'utf8', stdio: [ 'ignore', 'pipe', 'inherit' ], shell: true})
+    psql.stdout.on('data', function (data) {
+      result += data.toString()
+    })
+    psql.on('close', function (code) {
```

Comment:
@dickeyxxx I added an exit for non-zero codes in https://github.com/heroku/heroku-pg/pull/74/commits/d4b9c633041ca66bbcfa8bdc617c3414b32e5092 did you want me to change the `verifyExtensionsMatch` behavior or leave it alone?